src/domain/auth-rebac/domain/src/repos/rebac_repository.rs

```diff
@@ -115,6 +120,25 @@ pub struct EntityPropertyNotFoundError {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+#[derive(Error, Debug)]
+pub enum DeleteEntityPropertiesError {
+    #[error(transparent)]
+    NotFound(EntityNotFoundError),
```

Comment:
I would not expect to get an error if there are no properties to delete. Let's make APIs idempotent.

Reply:
The error type for `RebacService::delete_dataset_properties()` and `RebacService::delete_account_dataset_relation()` has been updated

Done

Reply:
NB. for the repository, the type remains the same -- idempotency is provided by the service